src/js/disability-benefits/utils/appeal-helpers.jsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
+import moment from 'moment';
 
-export const appealStatusDescriptions = (eventType) => {
-  const definitionMap = {
+export function appealStatusDescriptions(lastEvent, previousHistory = []) {
+  const contentMap = {
```

Comment:
I notice now that the `nextAction` objects require the arguments, though, so that's the only thing that would have to be specially handled.